main.js

```diff
@@ -0,0 +1,196 @@
+      reset();
+    } else if(svn == 'x' && eig == 'x' && nin == 'x'){
+      alert ("Player X WINS on turn " + count + "!!");
+      reset();
+    } else if(one == 'x' && four == 'x' && svn == 'x'){
+      alert ("Player X WINS on turn " + count + "!!");
+      reset();
+    } else if(two == 'x' && fiv == 'x' && eig == 'x'){
+      alert ("Player X WINS on turn " + count + "!!");
+      reset();
+    } else if(thr == 'x' && six == 'x' && nin == 'x'){
+      alert ("Player X WINS on turn " + count + "!!");
+      reset();
+    } else if(one == 'x' && fiv == 'x' && nin == 'x'){
+      alert ("Player X WINS on turn " + count + "!!");
+      reset();
+    } else if(svn == 'x' && fiv == 'x' && thr == 'x'){
+      alert ("Player X WINS on turn " + count + "!!");
+      reset();
+    } else if (one == 'o' && two == 'o' && thr == 'o') {
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(four == 'o' && fiv == 'o' && six == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(svn == 'o' && eig == 'o' && nin == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(one == 'o' && four == 'o' && svn == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(two == 'o' && fiv == 'o' && eig == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(thr == 'o' && six == 'o' && nin == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(one == 'o' && fiv == 'o' && nin == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if(svn == 'o' && fiv == 'o' && thr == 'o'){
+      alert ("Player O WINS on turn " + count + "!!");
+      reset();
+    } else if (count >= 10) {
+      alert ("Cat's Game! \nNeither X, nor O wins! \nBetter luck next game!");
+      reset();
+    }
+};
+
+var addX = function () {
+  var inputX = prompt("Player X, \nplease select an available number \nto place your x on the board!");
+    if (!(inputX > 0 && inputX < 10)){
+      console.error("Invalid selection. Please choose a number between 1 and 9");
+      addX();
+    }/* else if (inputX == 'o' || inputX == 'x'){
+      addX();
+    } */else {
+      if (inputX == 1 && one !== 'o') {
+        one = 'x';
+      } else if (inputX == 2 && two !== 'o'){
+        two = 'x';
+      }else if (inputX == 3 && thr !== 'o') {
+        thr = 'x';
+      }else if (inputX == 4 && four !== 'o') {
+        four = 'x';
+      }else if (inputX == 5 && fiv !== 'o') {
+        fiv = 'x';
+      }else if (inputX == 6 && six !== 'o') {
+        six = 'x';
+      }else if (inputX == 7 && svn !== 'o') {
+        svn = 'x';
+      }else if (inputX == 8 && eig !== 'o') {
+        eig = 'x';
+      }else if (inputX == 9 && nin !== 'o') {
+        nin = 'x';
+      }else {
+
+    }
+    checkWinner();
+    count++;
```

Comment:
Do you want to increment this every time here? What if there is an incorrect play? You have a bug where if there are incorrect plays, you game still ends the current game just based off this variable.